File: src/main/java/us/illyohs/civilmagiks/common/core/capability/ModCapabilities.java
package us.illyohs.civilmagiks.common.core.capability;

import net.minecraftforge.common.capabilities.CapabilityManager;
import us.illyohs.civilmagiks.api.capability.IManaHandler;
import us.illyohs.civilmagiks.common.core.capability.ManaStorage.ManaCallable;

public class ModCapabilities {

    public static void init() {
        CapabilityManager.INSTANCE.register(IManaHandler.class,  new ManaStorage(), new ManaCallable());
    }
}

File: src/main/scala/us/illyohs/civilmagiks/common/core/capability/ModCapabilities.scala
package us.illyohs.civilmagiks.common.core.capability

import net.minecraftforge.common.capabilities.CapabilityManager
import us.illyohs.civilmagiks.common.core.capability.ManaStorage.ManaCallable
import us.illyohs.civilmagiks.api.capability.IManaHandler

object ModCapabilities {
  def init = {
    CapabilityManager.INSTANCE.register(classOf[IManaHandler], new ManaStorage, new ManaCallable)
  }
}